core-bundle/src/Resources/contao/controllers/BackendPassword.php

```diff
@@ -127,6 +133,42 @@ public function run()
 
 		return $objTemplate->getResponse();
 	}
+
+	/**
+	 * Evaluate custom regex. See https://github.com/contao/contao/issues/5316
+	 *
+	 * @param  string      The password
+	 * @return bool|string The error message, false otherwise
+	 */
+	private function _evalRegexp($pw)
+	{
+		// no custom regex available
+		if (!isset($GLOBALS['TL_HOOKS']['addCustomRegexp']) || !\is_array($GLOBALS['TL_HOOKS']['addCustomRegexp']))
+			return false;
+
+		// load user data container
+		$this->loadDataContainer('tl_user');
+		$dc = new \DC_Table('tl_user');
```

Comment:
```suggestion
		$dc = new DC_Table('tl_user');
```

Reply:
Thanks, changed it. As https://github.com/contao/contao/actions/runs/3195462207/jobs/5216148107 states that `use Contao\DC_Table;` should not be used, I assume I should remove this line as well?

Reply:
Yes.